build.rs

```diff
@@ -53,40 +77,12 @@ fn main() -> Result<(), Box<dyn Error>> {
             .join("Cargo.toml");
     println!("cargo:warning=manifest_path={:?}", &manifest_path);
 
-    // Extract relevent features for the actors. This is far from perfect, bit it's "good enough"
-    // for what we're doing here.
-    let features = {
-        let mut cargo_toml: Table = toml::from_str(
-            &std::fs::read_to_string(&manifest_path).expect("failed to read Cargo.toml"),
-        )
-        .expect("failed to parse Cargo.toml");
-
-        if let Some(features_table) = cargo_toml.remove("features") {
-            let features_table: HashMap<String, Vec<String>> =
-                features_table.try_into().expect("failed to parse features table");
-
-            // Extract the features from the environment.
-            let features = std::env::vars_os()
-                .filter_map(|(key, _)| {
-                    key.to_str()
-                        .and_then(|k| k.strip_prefix("CARGO_FEATURE_"))
-                        .map(|k| k.to_owned())
-                })
-                .collect::<HashSet<_>>();
-
-            // Collect the transitive features. This is a best-effort operation because cargo messes
-            // with the feature names when it stores them in the environment, but it's good enough
-            // for our purposes here.
-            features_table
-                .into_iter()
-                .filter(|(k, _)| features.contains(&k.to_uppercase().replace('-', "_")))
-                .flat_map(|(_, v)| v)
-                .collect::<Vec<_>>()
-                .join(",")
-        } else {
-            String::new()
-        }
-    };
+    // Determine the network name.
+    let network_name = network_name();
+    println!("cargo:warning=network name: {}", network_name);
```

Comment:
What's the deal with these prints?

Reply:
It's the only way to get status information to the user. Otherwise, cargo will just sit there for a while the user is left wondering what's going on.

Reply:
ah ok